tests/integration/FtpClientTest.php

```diff
@@ -522,14 +522,17 @@ public function testCopyFromLocalWithFile() : void
         unlink($localFile);
     }
 
+    /**
+     * @depends testCopyToLocalWithDirectory
+    */
     public function testCopyToLocalWithFile() : void
     {
         $client = new FtpClient(ConnectionHelper::getConnection());
 
         $client->createFile(self::$testFile);
 
         $this->assertTrue($client->copyToLocal(self::$testFile, sys_get_temp_dir()));
```

Comment:
The `copyToLocal` method will use `trim` to remove the `/` character.

That is, when passing the `/tmp` folder and it will change into `tmp`. And they're different path.